sensor_msgs/CompressedImage image
aarapsi_robot_pack/ImageStructure data
std_msgs/Header header
